tests/TestCase/Cache/Engine/MemcachedEngineTest.php

```diff
@@ -18,9 +18,9 @@
 
 use Cake\Cache\Cache;
 use Cake\Cache\Engine\MemcachedEngine;
+use Cake\Cache\Exception\InvalidArgumentException;
 use Cake\TestSuite\TestCase;
 use DateInterval;
-use InvalidArgumentException;
```

Comment:
Looks like this test is using \InvalidArgumentException and shouldn't be changed.